### DISCUSSION TITLE
Tailwind Skeleton Generator 🦴✨

### DISCUSSION THREAD

**Discussion by fccoelho7** [tailwindlabs/tailwindcss#12104] [Show and tell]
🚀 Exciting News! Introducing Tailwind Skeleton Generator for Seamless Web Development! 🦴✨

Say goodbye to tedious skeleton screen development! Our AI-powered tool automates the creation of sleek skeleton loading components using Tailwind CSS 🤖💨

We value your input! Help us improve—try it out and share your feedback and suggestions.

🌟 Get started now: [skeletongenerator.com](https://www.skeletongenerator.com/)

**Comment by grtsnx** [comment]
<img width="951" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/49532326/a32dd0bd-ecee-4c47-8074-369253f155e5">

@fccoelho7

**Reply by edysegura** [reply]
It is back!! https://skeletongenerator.com/
cc: @thegrtnx

**Reply by Yathish26** [reply]
nope not there

**Reply by BenElferink** [reply]
![Screenshot 2025-01-31 at 18 52 11](https://github.com/user-attachments/assets/6e33b2bf-bd8b-4439-99bf-dfc420b2d5ba)

**Comment by grtsnx** [comment]
Thank you!

**Comment by Mat3uszkek** [comment]
its dead again

**Reply by Zvki1** [reply]
@Mat3uszkek  i think this link is working 
https://skeletongenerator.com/

**Reply by lylest** [reply]
site not reachable

**Comment by bcastillo-2022474** [comment]
Site not reachable... again

**Comment by AR-Tausif** [comment]
site not reachable. and also very loady. 
![image](https://github.com/user-attachments/assets/5adddc00-6ffa-41a8-877f-cf2fcc01bdcc)
